front/lib/api/assistant/generation.ts

```diff
@@ -382,6 +383,27 @@ export async function constructPrompt(
   return `${context}${instructions}`;
 }
 
+export function metaPromptForProvider(
+  providerId: ModelProviderIdType
+): string | null {
+  switch (providerId) {
```

Comment:
Are we sure this is provider-based and not model-based ? I would be tempted to store this on `ModelConfiguration`.

Reply:
Works for me !